docs-chef-io/content/inspec/resources/zfs.md

```diff
@@ -0,0 +1,82 @@
++++
+title = "zfs resource"
+draft = false
+gh_repo = "inspec"
+platform = "linux"
+
+[menu]
+  [menu.inspec]
+    title = "zfs"
+    identifier = "inspec/resources/os/zfs.md zfs resource"
+    parent = "inspec/resources/os"
++++
+
+Use the `zfs` Chef InSpec audit resource to test if the named ZFS Pool is present and/or has certain properties.
+
+
+## Availability
+
+### Installation
+
+This resource is distributed with Chef InSpec.
+
+## Syntax
+
+A `zfs` Chef InSpec audit resource allows to test if the named ZFS Pool is present and/or has certain properties.
```

Comment:
Why are we not giving reference to the third-party page as we provide in AWS resources?

Reply:
Reference for? @dkumaras

Reply:
zfs pool